connector/connect/common/src/main/buf.gen.yaml

```diff
@@ -16,21 +16,21 @@
 #
 version: v1
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/cpp:v3.20.0-1
+  - plugin: buf.build/protocolbuffers/cpp:v22.3
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.

Note, I've also bumped the versions to the **latest available**, but that may not be what's desired.